docs/technologies.md

```diff
@@ -2,6 +2,8 @@
 
 inserir imagem da arquitetura aqui
 
+https://drive.google.com/file/d/1CA24XPILXxBYLtid1PCO1mjSimEtloso/view?usp=sharing
```

Comment:
`![alt text](path/to/file)`
https://codigofonte.org/adicionar-imagens-ao-projeto-readme-no-github/

depois tenta por assim a imagem na docs

Reply:
Feito